src/components/DropdownMenu/DropdownMenu.stories.tsx

```diff
@@ -11,26 +11,28 @@ export default {
 
 const Template: ComponentStory<typeof DropdownMenu> = args => {
   return (
-    <DropdownMenu
-      {...args}
-      items={[
-        {
-          id: 'dropdown_menu_1',
-          label: 'Apple',
-          onSelect: () => console.log('you clicked Apple!')
-        },
-        {
-          id: 'dropdown_menu_2',
-          label: 'Orange',
-          onSelect: () => console.log('you clicked Orange!')
-        },
-        {
-          id: 'pear',
-          label: 'Pear',
-          onSelect: () => console.log('you clicked Pear!')
-        }
-      ]}
-    />
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh', width: '100%' }}>
```

Comment:
Wrapped in div to center the elements on the storybook page so we are able to see all alignments and side settings.